server/game/GameActions/MoveToBottomAction.js

```diff
@@ -1,6 +1,11 @@
 const CardGameAction = require('./CardGameAction');
 
 class MoveToBottomAction extends CardGameAction {
+    constructor(propertyFactory) {
+        super(propertyFactory);
+        this.targetPlayer = 'current';
```

Comment:
I think you should use a "player" attribute: See CaptureAction and UseAction. using "current" and "opponent" can be really tricky for example, if you the opponent triggers an effect, so, for consistenct, let's use player.

You could also assume that, since the cards are always coming from the deck, you could using card.owner instead of context.player to change the deck.